lib/gitlab/client/access_requests.rb

```diff
@@ -0,0 +1,96 @@
+class Gitlab::Client
+  # Defines methods related to Award Emojis.
+  # @see https://docs.gitlab.com/ce/api/access_requests.html
+  module AccessRequests
+    # Gets a list of access requests for a project viewable by the authenticated user.
+    #
+    # @example
+    #   Gitlab.project_access_requests(1)
+    #
+    # @param  [Integer, String] :project(required) The ID or name of a project.
+    # @return [Array<Gitlab::ObjectifiedHash>] List of project access requests
+    def project_access_requests(project)
+      get("/projects/#{url_encode project}/access_requests")
+    end
+    # Gets a list of access requests for a group viewable by the authenticated user.
+    #
+    # @example
+    #   Gitlab.group_access_requests(1)
+    #
+    # @param  [Integer, String] :group(required) The ID or name of a group.
+    # @return [Array<Gitlab::ObjectifiedHash>] List of group access requests
+    def group_access_requests(group)
+      get("/groups/#{url_encode group}/access_requests")
+    end
+    # Requests access for the authenticated user to a project.
+    #
+    # @example
+    #    Gitlab.request_project_access(1)
+    #
+    # @param  [Integer, String] :project(required) The ID or name of a project.
+    # @return <Gitlab::ObjectifiedHash] Information about the requested project access
+    def request_project_access(project)
+      post("/projects/#{url_encode project}/access_requests")
+    end
+    # Requests access for the authenticated user to a group.
+    #
+    # @example
+    #    Gitlab.request_group_access(1)
+    #
+    # @param  [Integer, String] :group(required) The ID or name of a group.
+    # @return <Gitlab::ObjectifiedHash] Information about the requested group access
+    def request_group_access(group)
+      post("/groups/#{url_encode group}/access_requests")
+    end
+    # Approves a project access request for the given user.
+    #
+    # @example
+    #    Gitlab.approve_project_access_request(1, 1)
+    #    Gitlab.approve_project_access_request(1, 1, 30)
+    #
+    # @param  [Integer, String] :project(required) The ID or name of a project.
+    # @param  [Integer] :user_id(required) The user ID of the access requester
+    # @param  [Integer] :access_level(optional) A valid access level (defaults: 30, developer access level)
+    # @return <Gitlab::ObjectifiedHash] Information about the approved project access request
+    def approve_project_access_request(project, user_id, access_level = '')
```

Comment:
What if the third argument were an `options` hash?
```
def approve_project_access_request(project, user_id, options = {})
```
Then the assignment on the next line isn't necessary. What do you think?